src/etcd/model/lease.cr

```diff
@@ -1,31 +1,44 @@
 require "./base"
 
 module Etcd::Model
-  struct Grant < Base
-    @[JSON::Field(key: "ID", converter: Etcd::Model::StringTypeConverter(Int64))]
-    getter id : Int64
-    @[JSON::Field(key: "TTL", converter: Etcd::Model::StringTypeConverter(Int64))]
-    getter ttl : Int64
+  struct LeasesArray < WithHeader
```

Comment:
I think `LeasesResponse` is better here or an alternative. It's not the `LeasesArray` in practice.

Reply:
`Leases`?